### PR TITLE
Replace golem.utils.encode_hex

### PR DESCRIPTION
**Pull request by Krigpl**
No need for our own implementation is there's already one existing.
NOTE: `eth_utils.encode_hex` prepends `0x` while `golem.utils` doesn't.

### CODE REVIEW

File: golem/model.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ class RawCharField(CharField):
     """ Char field without auto utf-8 encoding."""
 
     def db_value(self, value):
-        return str(encode_hex(value))
+        return str(encode_hex(value)[2:])
```
Comment (maaktweluit) [inline golem/model.py]: does it matter here if the `str(` cast is before or after the slice `[2:]`?

**Review by maaktweluit — APPROVED** [review]
LGTM!

Nice cleanup!

Small q, probably not trivial

File: golem/model.py (hunk 1 above)
Comment (Krigpl) [reply golem/model.py]: Hm, the `str` cast in unnecessary even

**Comment by etam** [comment]
Seems to be good, but tests are failing

**Comment by Krigpl** [comment]
> Seems to be good, but tests are failing

aka "the usual"

File: golem/model.py (hunk 1 above)
Comment (maaktweluit) [reply golem/model.py]: awesome! even better :D

**Review by etam — APPROVED** [review]
ok

**Pull request merged**